tornado_rest_client/test/test_api.py

```diff
@@ -165,6 +175,19 @@ def test_replace_path_tokens(self):
         self.assertEquals(str(ret), '/test/abcd/info')
 
     @testing.gen_test
+    def test_new_style_path_with_property_access(self):
+        test_consumer = RestConsumerTest(client=RestClientTest())
+
+        # the path with an arg must exist as an access method, and fail if you
+        # don't supply the arg.
+        with self.assertRaises(TypeError):
```

Comment:
That's the point of my comment. If you're testing for this behavior it means that's how you want it to act. So if someone tries to use this library with the old style of callables then TypeError looks like a bug in the code, rather than a change of convention.